Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/interoperability/InteroperabilityRepository.kt

```diff
@@ -0,0 +1,29 @@
+package de.rki.coronawarnapp.storage.interoperability
+
+import de.rki.coronawarnapp.service.applicationconfiguration.ApplicationConfigurationService
+import de.rki.coronawarnapp.storage.LocalData
+import kotlinx.coroutines.runBlocking
+import java.util.Locale
+
+object InteroperabilityRepository {
```

Comment:
Does not need to be an object, does it? IMHO classes should be preferred over objects and be injected via Dagger.

Reply:
I think you're right. Maybe I should change this to a more DI friendly file. 👍

Reply:
Change to new approach.